api/go.mod

```diff
@@ -1,10 +1,32 @@
 module github.com/ceph/ceph-csi/api
 
-go 1.16
+go 1.18
```

Comment:
I don't think we want to require Go 1.18 for projects that import this api. Please use an older supported version of Go.

Reply:
@nixpanic as mentioned in PR description , 1.16 and 1.17 are EOL . which version you are referring by "older supported version of Go" ?

Reply:
I had to search where to find those details. In future, include the link to the EOL notice for a version please, that makes things much easier to verify.